vpr/src/route/connection_router.cpp

```diff
@@ -248,7 +275,9 @@ std::vector<t_heap> ConnectionRouter<Heap>::timing_driven_find_all_shortest_path
 
     //Add the route tree to the heap with no specific target node
     int target_node = OPEN;
-    add_route_tree_to_heap(rt_root, target_node, cost_params);
+
+    // Don't need to run RCV when this is called, it's only called by the placer
+    add_route_tree_to_heap(rt_root, target_node, cost_params, false);
```

Comment:
When passing constants to arguments it is good style to document what the argument is like `/*argument_name=*/...`, because the typical context of the variable name is lost here.

Reply:
That sounds good, do you mean something like:
`add_route_tree_to_heap(rt_root, target_node, cost_params, /*run_rcv=*/ false);` ?

Reply:
Ya

Reply:
Still outstanding